viewpost.php

```diff
@@ -0,0 +1,58 @@
+<?php
+include ('config/init.php');
+$post =getPostInformation($_REQUEST['postID']);
+siteHeader('blog * '.$post["title"].'','blog + inspiration');
+
+
+if ($post['nextPost'] == NULL) {
+    print "
+    <div class='postSubHeader'>
```

Comment:
There are a lot of places like this where you're echoing html from php. I'd suggest paying more attention to the indenting just to make the code more readable. It doesn't have an impact on the actual functionality, but it can make your life easier when you come back to edit this later. It's also a much bigger deal if you ever work on code with other people, because they'll be used to reading html where child elements are indented.